Author: Stephen Gran <sgran@debian.org>
Description: Correct documentation about foreign key references
--- a/conf/dbschema.sql
+++ b/conf/dbschema.sql
@@ -26,8 +26,8 @@
 	"gid" int4 NOT NULL,
 	"uid" int4 NOT NULL,
 	PRIMARY KEY ("gid", "uid"),
-	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "groups"("gid"),
-	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "accounts"("uid")
+	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "group_table"("gid"),
+	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "passwd_table"("uid")
 );
   
 CREATE TABLE "shadow_table" (
--- a/doc/nss-pgsql.xml
+++ b/doc/nss-pgsql.xml
@@ -374,8 +374,8 @@
 	"gid" int4 NOT NULL,
 	"uid" int4 NOT NULL,
 	PRIMARY KEY ("gid", "uid"),
-	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "groups"("gid"),
-	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "accounts"("uid")
+	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "group_table"("gid"),
+	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "passwd_table"("uid")
 );
   
 CREATE TABLE "shadow_table" (
